lib/SimpleRenderer.js

```diff
@@ -0,0 +1,22 @@
+var fs = require('fs');
```

Comment:
What do you think about http://blog.heroku.com/archives/2015/11/10/node-habits-2016#4-stick-with-lowercase ?